src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.cs

```diff
@@ -189,7 +187,7 @@ protected override SyntaxToken GetMethodNameAtInvocation(IEnumerable<SyntaxNodeO
                 var currentType = semanticModel.GetSpeculativeTypeInfo(contextNode.SpanStart, typeName, SpeculativeBindingOption.BindAsTypeOrNamespace).Type;
                 if (currentType == null || !SymbolEqualityComparer.Default.Equals(currentType, semanticModel.ResolveType(typeParameter)))
                 {
-                    return new OperationStatus(OperationStatusFlag.BestEffort,
+                    return new OperationStatus(OperationStatusFlag.Succeeded,
```

Comment:
many (though nto all) of hte 'best effort' cases just go down the succeeded path.  User will be prompted if htey want to proceed with them in the command handler, or they will just see it happen in code actions.  This is the same behavior as before.

The ones that are slightly different are called out in teh review below.